#robot state, elements up to first order
Header header
float32 joint_position
float32 joint_velocity
float32 servo1_position
float32 servo1_velocity
float32 servo2_position
float32 servo2_velocity

